src/App.jsx

```diff
@@ -15,6 +16,17 @@ import { Notifications, setUserRoleStatuses } from 'src/libs/utils'
 import { extractError } from 'src/utils/ErrorUtils'
 import { Spinner } from 'src/components/Spinner'
 
+const queryClient = new QueryClient({
+  defaultOptions: {
+    queries: {
+      staleTime: 5 * 60 * 1000,
+      gcTime: 10 * 60 * 1000,
+      refetchOnWindowFocus: false,
+      retry: 1,
+    },
+  },
+})
```

Comment:
Since the Data Library does not frequently refresh the data, the settings are as follows:

* consider data fresh for 5 minutes
* remove unused data (when component dismounts) after 10 minutes
* do not refresh data on window focus
* allow one retry for data